### ISSUE TITLE
Key combinations from wtype / ydotool are not registered

### ISSUE THREAD

**Issue by t3e** [xournalpp/xournalpp#5670]
### Operating System

Linux

### (Linux only) Distribution

Debian

### (Linux only) Desktop Environment

Sway

### (Linux Only) Display Server

Wayland

### Installation Method

Debian Repo and Flatpak

### Xournal++ Version

1.2.3

### libgtk Version

3.24.41

### Bug Description

In my attempt to make the buttons on my wacom tablet work I tried these two tools. They seem to work on other programs but in xournalpp all combinations (with ctrl + xy) do not work for some reason.

### Expected Behaviour

Key combinations like Ctrl+Z should be picked up by xournalpp also from a virtual keyboard.

### Steps to Reproduce

wtype -M ctrl -k z

It's of course possible that it's a issue in wtype, but I have successfully tested such combinations with firefox, so I thought there's at least a good possibility that it's related to xournalpp and how it handles input events.

**Comment by tattsan** [comment]
If you are using software that captures your keystrokes, it may be recognized as one separate device. In my case this happened when I used `xkeysnail`. Open the `Input Systems` tab in Preferences and check if such a device appears in Input Devices. In my case it appeared as `py-evdev-uinput (mouse)`, and I configured it as `Mouse+Keyboard`.

**Comment by t3e** [comment]
I've just compiled the latest version but it has the same issue unfortunately.
I'm currently using wayland so xkeysnail is not an option for me. I also can't see any special input devices beeing detected when I use wtype.

**Comment by t3e** [comment]
Nope, unfortunately I had no success. I think I probably have to try a more recent version of ydotool. From what I see wtype looks somewhat abandoned currently in terms of development activity. I'm not sure if this is a general wayland problem because I once read it has issues with virtual input devices but I thought they are now mostly fixed. But anyway, it's probably not primarily an issue in xournal++ then...

**Comment by t3e** [comment]
Everything is now working with this tool:

https://git.sr.ht/~geb/dotool

Looks like wtype and ydotool are not very reliable at the moment.
